okpt/main.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+""" Runner script that serves as the main controller of the testing tool.
+"""
+
+import sys
+
+from okpt.io import reader, writer
+from okpt.io.config import validator
+
+
+def main():
+    """Main function of entry module."""
+    reader.define_args()
+    args = reader.get_args()
```

Comment:
Why not use named tuples?

Reply:
I've refactored args to use named tuples instead.